src/charm.py

```diff
@@ -479,6 +488,16 @@ def _create_internal_credentials(self) -> list[tuple[str, str]]:
 
         return credentials
 
+    def _set_os_config(self) -> None:
+        """Sets sysctl config."""
+        try:
+            self.sysctl_config.configure(OS_REQUIREMENTS)
+        except (sysctl.ApplyError, sysctl.ValidationError) as e:
+            logger.error(f"Error setting values on sysctl: {e.message}")
+            self._set_status(Status.SYSCONF_NOT_POSSIBLE)
+        except sysctl.CommandError:
+            logger.error("Error on sysctl")
```

Comment:
No sure but here it should also raise an exception to put charm in Error state or not?